platformio.ini

```
@@ -30,6 +30,12 @@ build_src_filter = +<*> -<.git/> -<.svn/>
 ; for version requirements see https://docs.platformio.org/en/stable/core/userguide/pkg/cmd_install.html#cmd-pkg-install-requirements
 lib_deps = 
   adafruit/Adafruit SSD1306@~2.5.3
+  lvgl@~8.3.7
```

We may also allow newer patch versions.

```suggestion
  lvgl@^8.3

```